client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -499,7 +503,24 @@ export const ApplicationsTable: React.FC = () => {
         ],
         getItemValue: (item) => normalizeRisk(item.risk) ?? "",
       },
+      {
+        categoryKey: "analysis",
+        title: t("terms.analysis"),
+        type: FilterType.multiselect,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.analysis").toLowerCase(),
+          }) + "...",
+        selectOptions: Array.from(taskStateToAnalyze).map(
+          ([taskState, displayStatus]) => ({
+            value: taskState,
+            label: t(`${displayStatus}`),
```

Comment:
The displayed values should match the values that the user sees in the table. Here the things get a bit tricky. 
Basically you need to repeat the same flow as done in the table by [ApplicationAnalysisStatus component](https://github.com/konveyor/tackle2-ui/blob/c04bb35390b3bd06e425dc8cead7cd92df2dd478/client/src/app/pages/applications/applications-table/applications-table.tsx#L929). There the received state is mapped via `taskStateToAnalyze`  BUT later on inside [IconedStatus](https://github.com/konveyor/tackle2-ui/blob/c04bb35390b3bd06e425dc8cead7cd92df2dd478/client/src/app/components/Icons/IconedStatus.tsx#L69)
it's mapped again - the `label` value is the value that the user sees.